### PR TITLE
Change cluster profile for ibmcloud MAC jobs

### PR DESCRIPTION
**Pull request by Shilpa-Gokul**

### CODE REVIEW

**Comment by Prajyot-Parab** [comment]
Post merge of https://github.com/openshift/ci-tools/pull/3904
fyi @prb112

**Comment by prb112** [comment]
/retest 
corresponding PR is merged https://github.com/openshift/ci-tools/pull/3911/files

File: ci-operator/config/openshift/multiarch/openshift-multiarch-master__nightly-4.15.yaml [hunk 1]
```diff
@@ -320,7 +320,7 @@ tests:
 - as: ocp-e2e-ibmcloud-ovn-multi-ppc64le
   cron: 20 7,17 * * *
   steps:
-    cluster_profile: ibmcloud-multi-ppc64le
```
Comment (prb112) [inline ci-operator/config/openshift/multiarch/openshift-multiarch-master__nightly-4.15.yaml:323]: this is the existing mac job... please don't change.

Comment (Shilpa-Gokul) [reply ci-operator/config/openshift/multiarch/openshift-multiarch-master__nightly-4.15.yaml]: Done, removed changes from 4.15 jobs

**Comment by prb112** [comment]
Hey @Shilpa-Gokul can you remove [WIP]? Then I think we can open the review request. Thansk, Paul

**Pull request merged**